src/GraphQl/Resolver/Factory/ResolverFactory.php

```diff
@@ -45,23 +51,28 @@ public function __invoke(string $resourceClass = null, string $rootClass = null,
                 return null;
             }
 
-            // Handles relay nodes
-            $operation ??= new Query();
+            return $this->process($source, $args, $info, $rootClass, $operation, null);
+        };
+    }
+
+    private function process(?array $source, array $args, ResolveInfo $info, string $rootClass = null, Operation $operation = null, mixed $body)
```

Comment:
Resolve instead of process? Since it will use the provider and the processor? Otherwise the PR seems good :slightly_smiling_face:

Reply:
Done